libraries/deploy-manager/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@rushstack/deploy-manager",
```

Comment:
Let's discuss the best name for this NPM package.

Reply:
@D4N14L and I  chatted and decided to call this package `@rushstack/package-extractor`.

This is partly based on feedback from @elliot-nelson that `rush deploy` could have a better name since it really isn't doing "deployments."  Eventually we should rename `rush deploy` to `rush extractor`.